language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/AbstractCompletionProvider.java

```diff
@@ -308,6 +308,9 @@ private List<LSCompletionItem> getTypeItems(BallerinaCompletionContext context)
                 completionItem.setInsertText(moduleName + ":" + insertText);
                 completionItem.setLabel(moduleName + ":" + label);
                 completionItems.add(lsCompletionItem);
+                if (completionItem.getFilterText() != null) {
+                    completionItem.setFilterText(moduleName + "_" + completionItem.getFilterText());
```

Comment:
Can't we set this filter text(i.e: filter text with module name) inside `getCompletionItemList` function(L:299)?

Reply:
No we can't because we only want to set the filter text for the types of external modules.